tests/main/system-usernames-microk8s/task.yaml

```diff
@@ -0,0 +1,70 @@
+summary: ensure only approved snaps can use the microk8s system user
+
+environment:
+  STORE_DIR: $(pwd)/fake-store-blobdir
+
+prepare: |
+    if [ "$TRUST_TEST_KEYS" = "false" ]; then
+        echo "This test needs test keys to be trusted"
+        exit
+    fi
+
+    echo "Ensure jq is installed"
+    if ! command -v jq; then
+        snap install --devmode jq
+    fi
+
+    echo "Ensure yaml2json is installed"
+    if ! command -v yaml2json; then
+        snap install --devmode remarshal
```

Comment:
:smile:

Reply:
I'm afraid I'm not getting the humour :-)

Is there something wrong or weird in this check?

Reply:
No I'm just happy to see someone else use remarshal in the tests, I maintain that snap